file/1.5/modules/ROOT/pages/file-acb-configuration.adoc

```diff
@@ -0,0 +1,246 @@
+= Using Anypoint Code Builder to Configure File Connector - Mule 4
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-connector-config-intro"]
+
+== Creating a New Integration Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-create-integration-project"]
+
+== Adding the Connector to Your Integration Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-add-connector-to-project"]
+
+[[configuring-trigger]]
+== Configuring the Trigger For the Flow
+
+A trigger (source) initiates a flow when a specified condition is met.
+You can configure one of these triggers to use with File Connector:
+
+* *On New or Upated File*
++
+Triggers a flow for a newly created or modified file.
+* *HTTP > Listener* 
++
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler* 
++
+Initiates a flow when a time-based condition is met.
++
+See xref:anypoint-code-builder::acb-component-scheduler.adoc[].
+
+For example, to configure the *On New or Updated* trigger:
+
+. Add the *On New or Updated* operation to the canvas UI.
+. Click the *On New or Updated* operation on the canvas UI. 
+. In the *General* tab, configure the following attributes:
++
+[%header%autowidth.spread]
```

Comment:
We can't reuse anything from references because references are generated. But maybe I (or you) can think this through and find a way. Otherwise, the only thing we are documenting in this topic is the stuff that appears in the ACB configuration UI, and the XML names don't align to the UI. I would love to revisit this later on.